### ISSUE TITLE
Cannot read properties of undefined (reading 'title')

### ISSUE THREAD

**Issue by wca4a** [tloncorp/landscape-apps#2596]
```
TypeError: Cannot read properties of undefined (reading 'title')
    at Ml (https://malmur-halmex.tlon.network/apps/groups/assets/index.45654692.js:48:183288)
    at Kl (https://malmur-halmex.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:52884)
    at mf (https://malmur-halmex.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:8723)
    at pf (https://malmur-halmex.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:947)
    at yg (https://malmur-halmex.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:875)
    at rn (https://malmur-halmex.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:730)
    at sl (https://malmur-halmex.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:21:10884)
    at https://malmur-halmex.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38716
    at e.unstable_runWithPriority (https://malmur-halmex.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:8:3811)
    at _r (https://malmur-halmex.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38494)
```

**Comment by patosullivan** [comment]
Hi, can you share:

a) what you were trying to do at the time (if anything)
b) the output of `+vats %groups, =verb &`

Thanks

**Comment by wca4a** [comment]
I was trying to leave a group

[image: image.png]

**Comment by ixv** [comment]
Just had the same issue, here is the output of that command
```
> +vats %groups, =verb &
%groups
  /sys/kelvin:      [%zuse 413]
  base hash:        0v1d.vhqub.fsoao.crbfe.gs49t.rhegj.govgq.g8ul5.frsbp.vv2tk.t68u4
  %cz hash:         0v1d.vhqub.fsoao.crbfe.gs49t.rhegj.govgq.g8ul5.frsbp.vv2tk.t68u4
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~sogryp-dister-dozzod-dozzod]
  updates:          remote
  source ship:      ~sogryp-dister-dozzod-dozzod
  source desk:      %groups
  source aeon:      30
  kids desk:        ~
  pending updates:  ~
```

**Comment by jamesacklin** [comment]
Fixed by #2635
